import-export-cli/integration/testutils/apiProduct_testUtils.go

```diff
@@ -0,0 +1,499 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shouldn't we mention the Year?

Example: [Here](https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/operationComponents/OperationGovernance.jsx#L2)

Reply:
@tmkasun In API Controller packages, we don't add the year along with the license header.